homeassistant/helpers/entity_platform.py

```diff
@@ -25,6 +25,7 @@
 SLOW_SETUP_MAX_WAIT = 60
 PLATFORM_NOT_READY_RETRIES = 10
 DATA_ENTITY_PLATFORM = "entity_platform"
+PLATFORM_NOT_READY_BASE_WAIT_TIME = 30  # seconds
```

Comment:
Let me know if this is ok. I saw other tests seemed to just add an arbitrary amount of time but wanted to be more exact and advance time exactly as much as one loop waited. To do that, I needed to make the wait time into a constant in `entity_platform.py`. I can undo this and just add like 5 minutes or something instead.

Reply:
Yeah, I don't think we should change this. At least not in this PR.

Reply:
It is convenient for tests to have to the constant though, so maybe suggest that in a separate PR.

Reply:
Ok sounds good, removed it from this PR